src/Component/Menu/User/User.less

```diff
@@ -3,6 +3,7 @@
   font-size: 1.2em;
   font-weight: bold;
   cursor: pointer;
+  margin-right: 20px;
```

Comment:
very minor: `margin` is most likely not a part of the component itself but should be set in the parent component. Maybe the Header?

Reply:
Fixed in 3538886.